util/process-doc/process-doc.go

```diff
@@ -159,23 +176,33 @@ func handleTypeAlias(item *ast.GenDecl, td *ast.TypeSpec) {
 	}
 }
 
+func extractDocs(ctx ast.Node, d *ast.CommentGroup, sep string) string {
+	if d == nil {
+		return ""
+	}
+
+	var docs string
+	var todo bool
+	for _, c := range d.List {
+		doc := c.Text
+		ndoc := strings.TrimPrefix(doc, "// ")
+		if ndoc == doc {
+			ndoc = strings.TrimPrefix(doc, "//")
+		}
+		if !todo && strings.Index(ndoc, "TODO") >= 0 {
+			todo = true
+		}
+		docs += ndoc + sep
+	}
+	return strings.Trim(docs, sep)
+}
+
 func printDoc(ctx ast.Node, d *ast.CommentGroup) {
 	if d == nil {
 		warn(ctx, "No documentation")
 	} else {
-		var todo bool
-		for _, c := range d.List {
-			doc := c.Text
-			ndoc := strings.TrimPrefix(doc, "// ")
-			if ndoc == doc {
-				ndoc = strings.TrimPrefix(doc, "//")
-			}
-			if !todo && strings.Index(ndoc, "TODO") >= 0 {
-				warn(ctx, "TODO in comment")
```

Comment:
This warning was lost in translation.

Reply:
indeed, added - also added the ndoc output to show the actual todo